tests/py35/test_autodoc_py35.py

```diff
@@ -207,11 +207,8 @@ def assert_order(items, objtype, name, member_order, **kw):
                    ('attribute', 'test_autodoc_py35.Class.inst_attr_inline'),
```

Comment:
This file should be merged into `tests/test_autodoc.py`. `py35` directory is no longer needed.

Reply:
done